rcl_logging_spdlog/CMakeLists.txt

```
@@ -20,6 +20,12 @@ find_package(spdlog REQUIRED)
 if(NOT WIN32)
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
+if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID MATCHES "Clang")
```

Since this is a C++ package, shouldn't this be:

```suggestion
if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_C_COMPILER_ID MATCHES "Clang")
```